backend/config/urls.py

```diff
@@ -35,6 +35,8 @@
     path('api/update_corpus_docs', views.update_corpus_docs),
     path('api/delete_corpus', views.delete_corpus),
     path('api/corpus/<int:corpus_id>', views.get_corpus),
+    path('api/corpus_from_csv/', views.create_corpus_csv),
+    path('api/upload_document', views.upload_document),
```

Comment:
Are both of these API endpoints in use? I _think_ I can see from the `views.py` file that `upload_document` is the endpoint we've settled on. If so, we can delete the second endpoint from this file.